tests/integration/test_logcollector/test_configuration/test_basic_configuration_alias.py

```diff
@@ -0,0 +1,161 @@
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#alias
+
+tags:
+    - logcollector_configuration
+'''
+import pytest
+import sys
+
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.constants.platforms import WINDOWS, MACOS, SOLARIS
+from wazuh_testing.modules.agentd import configuration as agentd_configuration
+from wazuh_testing.modules.logcollector import configuration as logcollector_configuration
+from wazuh_testing.modules.logcollector import patterns
+from wazuh_testing.modules.logcollector import utils
+from wazuh_testing.tools.monitors import file_monitor
+from wazuh_testing.utils import callbacks, configuration
+
+from . import TEST_CASES_PATH, CONFIGURATIONS_PATH
+
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# Variables
+local_internal_options = {logcollector_configuration.LOGCOLLECTOR_DEBUG: '2', logcollector_configuration.LOGCOLLECTOR_REMOTE_COMMANDS: '1', agentd_configuration.AGENTD_WINDOWS_DEBUG: '2'}
+
+if sys.platform == WINDOWS:
+    command = 'tasklist'
+    no_restart_windows_after_configuration_set = True
+elif sys.platform == MACOS:
+    command = 'ps aux'
+elif sys.platform == SOLARIS:
+    command = 'ps aux -xww'
+else:
+    command = 'ps -aux'
+
+# Test metadata, configuration and ids.
+cases_path = Path(TEST_CASES_PATH, 'cases_basic_configuration_alias.yaml')
+config_path = Path(CONFIGURATIONS_PATH, 'wazuh_basic_configuration.yaml')
+test_configuration, test_metadata, test_cases_ids = configuration.get_test_cases_data(cases_path)
+for test in test_metadata:
+    if test['command']:
+        test['command'] = command
+for test in test_configuration:
+    if test['COMMAND']:
+        test['COMMAND'] = command
+test_configuration = configuration.load_configuration_template(config_path, test_configuration, test_metadata)
+
+# Test daemons to restart.
+daemons_handler_configuration = {'all_daemons': True}
+
+# Test function.
+@pytest.mark.parametrize('test_configuration, test_metadata', zip(test_configuration, test_metadata), ids=test_cases_ids)
+def test_configuration_alias(test_configuration, test_metadata, configure_local_internal_options, truncate_monitored_files,
+                            set_wazuh_configuration, daemons_handler):
+    '''
+    description: Check if the 'wazuh-logcollector' daemon changes a command name in the log messages by
+                 the one defined in the 'alias' tag. For this purpose, the test will monitor a command
+                 using an alias. Then, it will verify that the 'reading command' event is generated.
+                 This event includes the output of the command executed and its alias. Finally, the test
+                 will verify that the Wazuh API returns the same values for the 'localfile' section that
+                 the configured one.
+
+    wazuh_min_version: 4.2.0
+
+    tier: 0
+
+    parameters:
```

Comment:
Fix test description here and in all tests where needed.

Reply:
Done